Documentation/Signoffs/jiegwuatu42-web_wireless-signoff.md

```diff
@@ -7,22 +7,36 @@ The Wireless acquisition of data helps effortlessly boost the quality of life. T
 
 The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner. Constraints on the subsystem are as follows:
 1. The server host machine must possess adequate storage space for the server code to store all collected data.
-2. The data will be encrypted with an undisclosed algorithm
+2. The data will be encrypted and decrypted with an undisclosed algorithm using AES Symmetric Encryption.  
+3. The server must be able to be established and connected to the internet.
+4. The server must have adequate storage for user data.
+5. The data received must be in legible format.
 
 
 ## BUILDABLE SCHEMATIC OF SUBSYSTEM
 
+<img width="350" alt="4G Connection" src="https://user-images.githubusercontent.com/110966922/219462337-5e72409b-17d1-4c59-8d68-eab86c8fda1f.png">
+<img width="432" alt="Wifi Connection" src="https://user-images.githubusercontent.com/110966922/219462340-b16763c0-cbe4-4803-ba0f-be77fbb212e4.png">
 
 
+Server will be made using HTML. It will receive info from TCP Connection provided from UART. 
 
 ## ANALYSIS OF CHOSEN COMPONENT(S)
 
-
+The team has chosen HTML5 UP to statisfy constraints and jobs of this system for the server/UI:
+1. Free web design templates that keeps price of the project affordable.
+2. Fully responsive software that allows easy between web and mobile platforms/
 
 ## BILL OF MATERIALS (BOM)
 
-| Designator | Manufacturer | Manufactured Part # | Description                                                   | Quantity | Price Each  |
-| ---------- | ------------ | ------------------- | ------------------------------------------------------------- | -------- | ----------  |
-|            |              |                     |                                                               |          |             |
-|            |              |                     |                                                               |          |             |
+N/A
```

Comment:
No parts??? Is there any hardware? What does the "server" consist of? Is there any software?

Reply:
How do the two listed components (both software) relate to the server. Is the server running on an Arduino?

Reply:
Provide URL to cherokee.

Show that license of cherokee is suitable for this project.

Reply:
Still need to address the software license for cherokee. Look up the license and in the analysis section state why it is acceptable for this project.

The parts list does not include any hardware. What will cherokee run on? List the server hardware, include it in the analysis, and include it in the parts list (but indicate quantity zero to order if the hardware is already available).